cattle/plugins/volmgr/handler.py

```diff
@@ -103,7 +103,11 @@ def _do_snapshot_backup(self, snapshot, storage_pool, progress):
                                blockstore_uuid)
         log.info("Backed up snapshot %s for volume %s" % (
             snapshot_uuid, volume_uuid))
-        snapshot.data.fields['backup'] = True
+        latest_snapshots_set = set([snapshot_uuid])
```

Comment:
What happens when you take 3 snapshots on a volume and they backup in parallel?  Is that possible?